src/blas/backends/cublas/cublas_batch.cpp

```diff
@@ -167,12 +167,25 @@ inline void gemm_batch_impl(sycl::queue &queue, transpose transa, transpose tran
             auto b_ = sc.get_mem<cuTypeB *>(b_acc);
             auto c_ = sc.get_mem<cuTypeC *>(c_acc);
             cublasStatus_t err;
-            CUBLAS_ERROR_FUNC_T_SYNC(
-                "cublasGemmStridedBatchedEx", cublasGemmStridedBatchedEx, err, handle,
-                get_cublas_operation(transa), get_cublas_operation(transb), m, n, k, &alpha, a_,
-                get_cublas_datatype<cuTypeA>(), lda, stride_a, b_, get_cublas_datatype<cuTypeB>(),
-                ldb, stride_b, &beta, c_, get_cublas_datatype<cuTypeC>(), ldc, stride_c, batch_size,
-                get_cublas_datatype<cuTypeS>(), cublas_gemm_algo);
+#ifdef SYCL_EXT_ONEAPI_ENQUEUE_NATIVE_COMMAND
```

Comment:
Can you use `cublas_native_named_func("cublasGemmStridedBatchedEx", ...)` to avoid the `#ifdef`? Here and in a few similar places below.

Reply:
Hi Andrew, thanks for the review, please see https://github.com/oneapi-src/oneMKL/pull/572/files#r1768732674 for the reasoning here.

Reply:
That link doesn't show me anything relevant, but if you have a reason I trust you, I just thought it might be an oversight.

Reply:
This is what I meant mainly 

"Yeah so for the few cases that I didn't use cublas_native_named_func this is because the functions don't pass the function name as a template parameter, so that I cannot use the generic function cublas_native_named_func."

"It seems that for some reason the compiler cannot infer the type of the cublas functions ending in *Ex. I don't know why that is, but seems that I cannot use the cublas_native_named_func in those cases. However I spotted that there was one case in this file where I should have used it and didn't, so I've just updated that."

Reply:
Got it, thanks! Sorry I missed that comment.

Reply:
Fixed the format now. Thanks